src/mixins/extensions/LoadFieldComponents.js

```diff
@@ -27,6 +27,33 @@ export default {
         }
       });
     },
+    createObjectURL(base64image) {
+      if (base64image) {
+        const binaryData = atob(base64image.split(",")[1]);
+        const binaryArray = new Uint8Array(binaryData.length);
+        for (let i = 0; i < binaryData.length; i++) {
+          binaryArray[i] = binaryData.charCodeAt(i);
+        }
+        const blob = new Blob([binaryArray], { type: "image/jpeg" });
```

Comment:
Hardcoding the extension file can be a double edge sword. Maybe we need to get the file extension using regex or whatever and use that value here. What do you think @caleeli

Reply:
Agree!